package.json

```diff
@@ -46,7 +46,8 @@
     "istanbul": "0.4.2",
     "mocha": "2.4.5",
     "nyc": "7.0.0",
-    "semantic-release": "^4.3.5"
+    "semantic-release": "^4.3.5",
+    "sinon": "1.17.4"
```

Comment:
So Sinon helps us to 'watch' / 'fake' methods in our code. I explained what it does with these 2 examples:
[Comment #1](https://github.com/andresdavid90/pizza-guy/pull/13/files#r71974311)
[Comment #2](https://github.com/andresdavid90/pizza-guy/pull/13/files#r71974322)